examples/tutorials/text.py

```diff
@@ -152,4 +152,4 @@
 #
 # For crafting more advanced styles, be sure to check out the GMT documentation
 # at :gmt-docs:`text.html` and also the cookbook at
-# :gmt-docs:`cookbook/features.html#placement-of-text`. Good luck!
+# :gmt-docs:`PostScript Fonts Used by GMT <cookbook/postscript-fonts.html>`. Good luck!
```

Comment:
You made wrong changes here. I meant to change the reference at line 55, not here.

Reply:
Fixed in ec4e401.